pkg/ec2/getSG.go

```diff
@@ -0,0 +1,42 @@
+package ec2
+
+import (
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/ec2"
+	"github.com/spf13/cobra"
+)
+
+
+
+type SECURITYGRP struct {
```

Comment:
use camel case and split the security group and ssh in two different PRs.

each PR should have only one feature command. it's easy to review and focus.

Reply:
Okay